_posts/2018-07-19-futures-0.3.0-alpha.1.markdown

```diff
@@ -35,18 +35,34 @@ In other words:
 
 With futures 0.3 alpha, it is possible to use the new `async`/`await` notation that recently landed in nightly:
 
+```toml
+cargo-features = ["edition"]
+
+[package]
+edition = '2018
+```
+
 ```rust
+#![feature(async_await)]
+#![feature(await_macro)]
+
+extern crate futures;
+
 use futures::executor::block_on;
 use futures::future;
+use futures::prelude::*;
```

Comment:
```rust
use futures::future::{self, FutureExt);
```
should do the trick.

We normally try to list the dependencies. The `prelude` is great, but we normally use it only to hack around. Code examples should use normal imports.